blueprints/ember-frost-list/index.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-theme', target: '^1.3.0'}
+        {name: 'ember-frost-core', target: '0.0.13 - 1'}
```

Comment:
Ah, I see, as @sandersky pointed me to in Slack, https://github.com/npm/node-semver#hyphen-ranges-xyz---abc means that `0.0.13 - 1` is equivalent to `>=0.0.13 <2.0.0`, but I'd argue the latter is more clear, especially since the hyphen seems so similar to the prerelease identifier for something like `0.0.13-alpha-2`, etc.